src/ts/components/seekbar.ts

```diff
@@ -349,7 +349,10 @@ export class SeekBar extends Component<SeekBarConfig> {
       const maxTimeShift = this.player.getMaxTimeShift();
       this.player.timeShift(maxTimeShift - (maxTimeShift * (percentage / 100)), 'ui');
     } else {
-      this.player.seek(this.player.getDuration() * (percentage / 100), 'ui');
+      const seekableRangeStart = this.player.getSeekableRange() && this.player.getSeekableRange().start || 0;
```

Comment:
Did you ever experienced `this.player.getSeekableRange()` to be `null`/`undefined`?

Also, as we have 3 occurrences of this line we might consider putting it into ulils?

Reply:
When using it together with our mobile SDKs.

Reply:
c667ec2